### PR TITLE
Created a basic layout for About Shabd App

### PR DESCRIPTION
**Pull request by aparsh**
Fixes #81 
image

### CODE REVIEW

**Comment by atm1504** [comment]
Looks good to me. @amangrobo what about you?

File: app/src/main/res/layout/about_shabd.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="100dp"
```
Comment (atm1504) [inline app/src/main/res/layout/about_shabd.xml:8]: Remove all harxcoded values. Use dimens, colours, strings value.

**Review by atm1504 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/about_shabd.xml (hunk 1 above)
Comment (amangrobo) [reply app/src/main/res/layout/about_shabd.xml:8]: dimensions are OK for now but the text should be taken from resources

File: app/src/main/res/layout/about_shabd.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="25dp"
+        android:background="@drawable/rounded_edittext">
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Developer"
```
Comment (amangrobo) [inline app/src/main/res/layout/about_shabd.xml:55]: Developer, Overview and other titles should be taken from string resources

**Review by amangrobo — CHANGES_REQUESTED** [review]

**Comment by aparsh** [comment]
> Looks good to me. @amangrobo what about you?

Already such gradient file is being used in the app.
I can do it for all files later on.

File: app/src/main/res/layout/about_shabd.xml (hunk 2 above)
Comment (aparsh) [reply app/src/main/res/layout/about_shabd.xml:55]: making those changes

**Review by amangrobo — APPROVED** [review]

**Review by chanmol1999 — APPROVED** [review]

**Pull request closed without merging**